modules/rational-persistence.el

```diff
@@ -0,0 +1,35 @@
+;;; rational-persistence.el -*- lexical-binding: t; -*-
+
+;; Copyright (C) 2022
+;; SPDX-License-Identifier: MIT
+
+;; Author: System Crafters Community
+
+;; Commentary
+
+;; Some settings to make emacs more of a continous expierience.
+
+;;; Code:
+
+;; When you visit a file, point goes to the last place it was when file was previously visited.
+(save-place-mode 1)
+
+;; Saving of history (minibuffer, kill-ring, search-ring, keyboard macros, shell commands).
+(setq savehist-save-minibuffer-history t)
+  (setq savehist-additional-variables
+        '(kill-ring
```

Comment:
It's probably better not to include the kill ring because clipboard copies might contain sensitive information.

Reply:
Maybe the whole value of `savehist-additional-variables` should be avoided. We could document it as something a user _should_ configure in their own configuration.